src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -641,7 +693,9 @@ private Bid toBid(BidderBid bidderBid, String bidder, ExtRequestTargeting target
         final Video storedVideo = impIdToStoredVideo.get(bid.getImpid());
         final Events events = eventsEnabled ? eventsService.createEvent(bid.getId(), account.getId()) : null;
```

Comment:
When creating event URLs and VAST IMP URLs, use this generated bidid instead of the bidder-created id. 

Just one line above.....

Reply:
?